dm_cli/command_group/data_source.py

```diff
@@ -79,22 +79,25 @@ def initialize_data_source(path: Path):
         print(emoji.emojize(f"\t:warning: No data source definitions were found in '{data_sources_dir}'."))
 
     for data_source_definition_filename in data_source_definitions:
-        data_source_definition_filepath = Path(data_sources_dir).joinpath(data_source_definition_filename)
-        data_source_name = data_source_definition_filename.replace(".json", "")
-
-        data_source_data_dir = data_dir / data_source_name
-        if not data_source_data_dir.is_dir():
-            print(
-                emoji.emojize(
-                    f"\t:warning: No data source data directory was found by the name '{data_source_name}' in '{data_dir}'."
-                )
-            )
-            continue
+        import_data_source_file(data_sources_dir, data_dir, data_source_definition_filename)
+
+
+def import_data_source_file(data_sources_dir: str, data_dir: str, data_source_definition_filename: str):
```

Comment:
This is wrongly named. It seems to "import data_source definition and data source data".

Not sure this needs to be a separate function. Could just fix the bug in the old for loop.